src/common/util/contentSecurityPolicy.ts

```diff
@@ -0,0 +1,53 @@
+export const SECURITY_POLICY = {
+    directives: {
+        connectSrc: [
+            "'self' blob: data:",
+            '*.gov.uk',
+            'dc.services.visualstudio.com',
+            '*.launchdarkly.com',
+            'https://*.google-analytics.com',
+            'https://*.googletagmanager.com',
+            'https://*.analytics.google.com',
+            '*.hmcts.net',
+            'wss://*.webpubsub.azure.com',
+            'https://*.in.applicationinsights.azure.com',
+            'https://*.monitor.azure.com',
+        ],
+        defaultSrc: ["'self'"],
+        fontSrc: ["'self'", 'https://fonts.gstatic.com', 'data:'],
+        formAction: ["'none'"],
+        frameAncestors: ["'self'"],
+        frameSrc: ["'self'"],
+        imgSrc: [
+            "'self'",
+            'data:',
+            'https://*.google-analytics.com',
+            'https://*.googletagmanager.com',
+            'https://raw.githubusercontent.com/hmcts/',
+            'https://stats.g.doubleclick.net/',
+            'https://ssl.gstatic.com/',
+            'https://www.gstatic.com/',
+            'https://fonts.gstatic.com',
+        ],
+        mediaSrc: ["'self'"],
+        scriptSrc: [
+            "'self'",
+            "'unsafe-inline'",
+            "'unsafe-eval'",
+            'https://*.google-analytics.com',
+            'https://*.googletagmanager.com',
+            'az416426.vo.msecnd.net',
+        ],
+        styleSrc: [
+            "'self'",
+            "'unsafe-inline'",
+            'https://fonts.googleapis.com',
+            'https://fonts.gstatic.com',
+            'https://www.googletagmanager.com',
+        ],
+    },
+}
+
+export const getContentSecurityPolicy = (helmet: any) => {
```

Comment:
Can we not provide a type for the helmet object?

Reply:
It's more of an import than an object. Typing it is going to require further thought, especially as it is in a separate library now. Could add this as tech debt? Or try and add a type here that I could try to define the imported helmet as?

Reply:
If there's no easy way to import a type then it's fine as it is